tests/test_django/app/models.py

```
@@ -178,10 +195,10 @@ def __str__(self) -> str:
         return f"{self.book.title} ({self.inventory_number})"
 
 
-class Borrowing(models.Model):
+class Borrowing(BaseModel):
     """Model for a borrowing."""
 
-    user = models.ForeignKey(User, on_delete=models.CASCADE)  # Foreign Key relationship
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="user_borrowings")  # Foreign Key relationship
```

Ruff (0.8.2)

Line too long (120 > 88)

(E501)